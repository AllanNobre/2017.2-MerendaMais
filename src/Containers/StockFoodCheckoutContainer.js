import { connect } from 'react-redux';
import StockFoodCheckoutScreen from '../screens/StockFoodCheckoutScreen';
import setStockFoodReport from '../actions/reportActions';

const mapStateToProps = (state) => {
  console.log('StockFoodCheckoutContainer: ');
  console.log(state.report);

  return {
    report: state.report.foodStock,
  };
};

const mapDispatchToProps = dispatch => ({
  setStockFoodReport: key => dispatch(setStockFoodReport(key)),
});

const StockFoodCheckoutContainer =
connect(mapStateToProps, mapDispatchToProps)(StockFoodCheckoutScreen);

export default StockFoodCheckoutContainer;
