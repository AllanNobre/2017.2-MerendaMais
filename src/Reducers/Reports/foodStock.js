const foodStock = [
  { label: 'Escola não possui Estoque de Alimentos', marked: false, key: 0 },
  { label: 'Alimentos vencidos', marked: false, key: 1 },
  { label: 'Embalagens roídas', marked: false, key: 2 },
  { label: 'Insetos dentro das embalagens', marked: false, key: 3 },
  { label: 'Alimentos em contato com o chão', marked: false, key: 4 },
  { label: 'Alimentos dentro de caixas', marked: false, key: 5 },
  { label: 'Produtos de limpeza junto aos alimentos', marked: false, key: 6 },
  { label: 'Embalagens abertas', marked: false, key: 7 },
  { label: 'Prateleiras enferrujadas', marked: false, key: 8 },
  { label: 'Prateleiras encostadas na parede', marked: false, key: 9 },
  { label: 'Local abafado', marked: false, key: 10 },
  { label: 'Contato direto dos raios de sol com as embalagens', marked: false, key: 11 },
  { label: 'Falta de controle de entrada de alimentos', marked: false, key: 12 },
  { label: 'Falta de controle de saída de alimentos', marked: false, key: 13 },
  { label: 'Telhado sem forro', marked: false, key: 14 },
  { label: 'Telhado com furos', marked: false, key: 15 },
  { label: 'Congelador sem termômetro', marked: false, key: 16 },
  { label: 'Lixeiras destampadas', marked: false, key: 17 },
  { label: 'Chão difícil de limpar', marked: false, key: 18 },
  { label: 'Parede difícil de limpar', marked: false, key: 19 },
  { label: 'Janelas sem tela de proteção', marked: false, key: 20 },
  { label: 'Portas sem tela de proteção', marked: false, key: 21 },
  { label: 'Portas sem rodapé', marked: false, key: 22 },
  { label: 'Insetos sobrevoando o local', marked: false, key: 23 },
  { label: 'Animais com livre circulação', marked: false, key: 24 },
  { label: 'Ventilador de teto empoeirado', marked: false, key: 25 },
  { label: 'Equipamentos sujos', marked: false, key: 26 },
  { label: 'Local mal iluminado', marked: false, key: 27 },
  { label: 'Local sujo', marked: false, key: 28 },
  { label: 'Objetos que não sejam alimentos', marked: false, key: 29 },
  { label: 'Lixo não é armazenado em local fechado', marked: false, key: 30 },
];

export default foodStock;
