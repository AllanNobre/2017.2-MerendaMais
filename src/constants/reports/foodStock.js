const foodStock = [
  { label: 'Escola possui Estoque de Alimentos', markedYes: false, markedNo: false, status: false, key: 0 },
  { label: 'Alimentos vencidos', markedYes: false, markedNo: false, status: false, key: 1 },
  { label: 'Embalagens roídas', markedYes: false, markedNo: false, status: false, key: 2 },
  { label: 'Insetos dentro das embalagens', markedYes: false, markedNo: false, status: false, key: 3 },
  { label: 'Alimentos em contato com o chão', markedYes: false, markedNo: false, status: false, key: 4 },
  { label: 'Alimentos dentro de caixas', markedYes: false, markedNo: false, status: false, key: 5 },
  { label: 'Produtos de limpeza junto aos alimentos', markedYes: false, markedNo: false, status: false, key: 6 },
  { label: 'Embalagens abertas', markedYes: false, markedNo: false, status: false, key: 7 },
  { label: 'Prateleiras enferrujadas', markedYes: false, markedNo: false, status: false, key: 8 },
  { label: 'Prateleiras encostadas na parede', markedYes: false, markedNo: false, status: false, key: 9 },
  { label: 'Local abafado', markedYes: false, markedNo: false, status: false, key: 10 },
  { label: 'Contato direto dos raios de sol com as embalagens', markedYes: false, markedNo: false, status: false, key: 11 },
  { label: 'Falta de controle de entrada de alimentos', markedYes: false, markedNo: false, status: false, key: 12 },
  { label: 'Falta de controle de saída de alimentos', markedYes: false, markedNo: false, status: false, key: 13 },
  { label: 'Telhado sem forro', markedYes: false, markedNo: false, status: false, key: 14 },
  { label: 'Telhado com furos', markedYes: false, markedNo: false, status: false, key: 15 },
  { label: 'Congelador sem termômetro', markedYes: false, markedNo: false, status: false, key: 16 },
  { label: 'Lixeiras destampadas', markedYes: false, markedNo: false, status: false, key: 17 },
  { label: 'Chão difícil de limpar', markedYes: false, markedNo: false, status: false, key: 18 },
  { label: 'Parede difícil de limpar', markedYes: false, markedNo: false, status: false, key: 19 },
  { label: 'Janelas sem tela de proteção', markedYes: false, markedNo: false, status: false, key: 20 },
  { label: 'Portas sem tela de proteção', markedYes: false, markedNo: false, status: false, key: 21 },
  { label: 'Portas sem rodapé', markedYes: false, markedNo: false, status: false, key: 22 },
  { label: 'Insetos sobrevoando o local', markedYes: false, markedNo: false, status: false, key: 23 },
  { label: 'Animais com livre circulação', markedYes: false, markedNo: false, status: false, key: 24 },
  { label: 'Ventilador de teto empoeirado', markedYes: false, markedNo: false, status: false, key: 25 },
  { label: 'Equipamentos sujos', markedYes: false, markedNo: false, status: false, key: 26 },
  { label: 'Local mal iluminado', markedYes: false, markedNo: false, status: false, key: 27 },
  { label: 'Local sujo', markedYes: false, markedNo: false, status: false, key: 28 },
  { label: 'Objetos que não sejam alimentos', markedYes: false, markedNo: false, status: false, key: 29 },
  { label: 'Lixo não é armazenado em local fechado', markedYes: false, markedNo: false, status: false, key: 30 },
];

export default foodStock;
