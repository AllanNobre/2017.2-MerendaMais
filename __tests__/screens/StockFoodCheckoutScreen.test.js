import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import configureStore from 'redux-mock-store';
// imported as a connected component!
// import StockFoodCheckoutScreen from '../../src/screens/StockFoodCheckoutScreen'
import StockFoodCheckoutContainer from '../../src/Containers/StockFoodCheckoutContainer';
import foodStock from '../../src/Reducers/Reports/foodStock';
import StockFoodCheckoutScreen from '../../src/screens/StockFoodCheckoutScreen';

Enzyme.configure({ adapter: new Adapter() });

const mockStore = configureStore();

const initialState = {
  report: {
    foodStock,
    foodStockObservation: '',
  },
};

const store = mockStore(initialState);

describe('Testing StockFoodCheckoutScreen', () => {
  it('renders as expected', () => {
    const wrapper = shallow(
      <StockFoodCheckoutContainer />,
      { context: { store } },
    ).dive();
    expect(wrapper).toMatchSnapshot();
  });
});
