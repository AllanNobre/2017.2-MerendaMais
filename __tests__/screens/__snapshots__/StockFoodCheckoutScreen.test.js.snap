// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing StockFoodCheckoutScreen renders as expected 1`] = `
<ScrollViewMock
  style={
    Object {
      "flex": 6,
      "flexDirection": "column",
      "marginBottom": 9,
    }
  }
>
  <Header
    backButton={true}
    subTitle="Estoque de alimento"
    title="Relatório"
  />
  <View>
    <View
      key="0"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Escola não possui Estoque de Alimentos
      </Text>
    </View>
    <View
      key="1"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Alimentos vencidos
      </Text>
    </View>
    <View
      key="2"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Embalagens roídas
      </Text>
    </View>
    <View
      key="3"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Insetos dentro das embalagens
      </Text>
    </View>
    <View
      key="4"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Alimentos em contato com o chão
      </Text>
    </View>
    <View
      key="5"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Alimentos dentro de caixas
      </Text>
    </View>
    <View
      key="6"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Produtos de limpeza junto aos alimentos
      </Text>
    </View>
    <View
      key="7"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Embalagens abertas
      </Text>
    </View>
    <View
      key="8"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Prateleiras enferrujadas
      </Text>
    </View>
    <View
      key="9"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Prateleiras encostadas na parede
      </Text>
    </View>
    <View
      key="10"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Local abafado
      </Text>
    </View>
    <View
      key="11"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Contato direto dos raios de sol com as embalagens
      </Text>
    </View>
    <View
      key="12"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Falta de controle de entrada de alimentos
      </Text>
    </View>
    <View
      key="13"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Falta de controle de saída de alimentos
      </Text>
    </View>
    <View
      key="14"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Telhado sem forro
      </Text>
    </View>
    <View
      key="15"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Telhado com furos
      </Text>
    </View>
    <View
      key="16"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Congelador sem termômetro
      </Text>
    </View>
    <View
      key="17"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Lixeiras destampadas
      </Text>
    </View>
    <View
      key="18"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Chão difícil de limpar
      </Text>
    </View>
    <View
      key="19"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Parede difícil de limpar
      </Text>
    </View>
    <View
      key="20"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Janelas sem tela de proteção
      </Text>
    </View>
    <View
      key="21"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Portas sem tela de proteção
      </Text>
    </View>
    <View
      key="22"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Portas sem rodapé
      </Text>
    </View>
    <View
      key="23"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Insetos sobrevoando o local
      </Text>
    </View>
    <View
      key="24"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Animais com livre circulação
      </Text>
    </View>
    <View
      key="25"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Ventilador de teto empoeirado
      </Text>
    </View>
    <View
      key="26"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Equipamentos sujos
      </Text>
    </View>
    <View
      key="27"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Local mal iluminado
      </Text>
    </View>
    <View
      key="28"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Local sujo
      </Text>
    </View>
    <View
      key="29"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Objetos que não sejam alimentos
      </Text>
    </View>
    <View
      key="30"
      style={
        Object {
          "flexDirection": "row",
        }
      }
    >
      <CheckBox
        checked={null}
        checkedImage={1}
        containerStyle={
          Object {
            "flex": 1,
            "flexWrap": "wrap",
            "paddingLeft": 10,
            "paddingTop": 10,
          }
        }
        label=" "
        labelBefore={false}
        labelLines={1}
        onChange={[Function]}
        uncheckedImage={1}
        underlayColor="white"
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        disabled={false}
        ellipsizeMode="tail"
        style={
          Object {
            "flex": 1,
            "paddingTop": 15,
          }
        }
      >
        Lixo não é armazenado em local fechado
      </Text>
    </View>
  </View>
  <KeyboardAvoidingView
    behavior="padding"
    keyboardVerticalOffset={0}
  >
    <View
      style={
        Object {
          "paddingLeft": 10,
          "paddingRight": 10,
          "paddingTop": 30,
        }
      }
    >
      <TextInput
        multiline={true}
        onChangeText={[Function]}
        placeholder="Observações (opcional)"
        style={
          Object {
            "borderRadius": 12,
            "borderWidth": 1,
            "fontSize": 37.5,
            "height": 333.5,
            "paddingLeft": 10,
            "textAlignVertical": "top",
          }
        }
        underlineColorAndroid="transparent"
      />
    </View>
  </KeyboardAvoidingView>
  <TouchableOpacity
    activeOpacity={0.2}
    key="setObservation"
    onPress={[Function]}
    style={
      Object {
        "backgroundColor": "#FF9500",
        "borderRadius": 7,
        "borderWidth": 1,
        "justifyContent": "flex-end",
        "marginBottom": 20,
        "marginHorizontal": 15,
        "marginTop": 30,
        "paddingVertical": 10,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={
        Object {
          "color": "#FFF",
          "textAlign": "center",
        }
      }
    >
      Concluir
    </Text>
  </TouchableOpacity>
</ScrollViewMock>
`;
